release.nix

```diff
@@ -1,4 +1,4 @@
-{ runTest ? true, gitDescribe ? "TEST-dirty" }:
+{ runTest ? false, gitDescribe ? "TEST-dirty" }:
```

Comment:
Since `true` breaks the build right now, let's add a comment explaining why (ideally linking to an upstream issue, if there is one)

Reply:
Comment added, did not find an issue but asked gre-ledger if he can provide one.